Makefile

```diff
@@ -320,25 +320,11 @@ generate-e2e-templates: $(KUSTOMIZE) ## Generate e2e cluster templates
 
 .PHONY: lint
 lint: $(GOLANGCI_LINT) ## Lint the codebase
```

Comment:
If I see correctly this aligns the entire lint thing to core CAPI (which sounds good to me)

Reply:
Jep, did a diff for it!